content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -216,6 +216,24 @@ Used on: Service
 
 Kubernetes uses this label to differentiate multiple Services. Used currently for `ELB`(Elastic Load Balancer) only.
 
+### kubernetes.io/service-account.name
+
+Example: `kubernetes.io/service-account.name: "sa-name"`
+
+Used on: Secret
+
+This annotation records the {{< glossary_tooltip term_id="name" text="name">}} of the
+ServiceAccount that the token (stored in the Secret) represents.
```

Comment:
Hi @sftim @liggitt, "ServiceAccount that the token (stored in the Secret of type `kubernetes.io/service-account-token` ) represents"  make sense?